### ISSUE TITLE
Understanding .vbkm

### ISSUE THREAD

**Issue by GitHubRulesOK** [sumatrapdfreader/sumatrapdf#1357]
The Virtual bookmarks looks promising as a replacement for my current usage of favorites

Pros
Having a user defined grouping of multiple related files (thus can be used for a library of topics)
External to settings.txt thus should be more robust during update/shutdown BUT currently file states are still fragile as they are kept in settings.txt
Nice if it could rememember last Position for each sub file part ? (they are kept in settings.txt but overiden by the .vbkm re-selection)
![image](https://user-images.githubusercontent.com/12754913/71586615-c43d9f80-2b12-11ea-807f-9279904c956a.png)

Cons
1) Too many files with long contents list can be dificult to manage with expand ALL or collapse ALL
    context options to expand HERE and Collapse HERE would be very usefull
2) Only works on .pdf with contents (at present ?)
3) Can get confused by pdf containing attachments as contents (accepted but not shown)

In summary I have stated elsewhere it could be beneficial to migrate multiple file states (usage profiles / history etc) to a file external to settings.txt and the intent of using one or more .vbkm lends itself naturally to that purpose.
The complexity of adding and deleting bookmarks (and or annotation) per file lends itself to seperate editable .bkm files per pdf epub cbr etc.

**Comment by GitHubRulesOK** [comment]
redundant unsupported experimental feature
